src/System.Net.Security/src/System/IO/StreamAsyncHelper.cs

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+using System.Diagnostics.Contracts;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.IO
+{
+    /// <summary>Provides support for implementing asynchronous operations on Streams.</summary>
+    internal sealed class StreamAsyncHelper
+    {
```

Comment:
Why is the `StreamAsyncHelper` class buried in this dll?  Why not put into src/Common somewhere? Then it could be reused by others.